src/libs/runtime-utils.js

```diff
@@ -222,33 +223,37 @@ export const machineCost = machineType => {
   return _.find(knownMachineType => knownMachineType.name === machineType, machineTypes).price
 }
 
-export const getCurrentApp = _.flow(trimAppsOldestFirst, _.last)
+export const getCurrentAppForType = appType => _.flow(trimAppsOldestFirst, _.remove(_ => _.appType !== appType), _.last)
```

Comment:
Is `_.remove(badPredicate)` a negative rephrasing of an alternative `_.filter(positivePredicate)`? If so, for readability maybe the positive predicate version would be easier for humans to parse in this case?